### DISCUSSION TITLE
Force VS2022 to use opencover instead of coverlet

### DISCUSSION THREAD

**Discussion by rajeshpa** [FortuneN/FineCodeCoverage#320] [Q&A]
Hello,

I have a solution running in VS 2022 and I am using opencover for coverage. I am also using FCC to see the coverage results. By default when I run the tests, coverlet is being picked up as the coverage tool. Is there any option to force opencover instead of coverlet as the coverage tool ? 

Thanks in advance

**Comment by tonyhallett** [comment]
No. It is solely on determined by the project style. SDK style projects go to coverlet as it supports Net Core. 
Opencover 
Applications targetting .Net Core may encounter issues but using the -oldstyle switch usually works.

Given the "usually" and the fact it has been archived this choice seems sensible. If you open an issue with this feature request it can be implemented in the future.

**Reply by rajeshpa** [reply]
Ok.. Thanks
